doc/python_api/rst/bge_types/bge.types.KX_Scene.rst

```diff
@@ -143,10 +143,15 @@ base class --- :class:`PyObjectPlus`
 
       :type: Vector((gx, gy, gz))
 
-   .. attribute:: lodGlobalScale
+   .. method:: setLodScale(pythonObjectList, scale)
```

Comment:
Is not too much faster than that in C++ because you still iterate in a python list, yes you avoid time on `lodManager.distanceScale`, but is very insignificant.